tools/dockerfile/grpc_artifact_python_manylinux2014_aarch64/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM dockcross/manylinux2014-aarch64:20210826-19322ba
+FROM dockcross/manylinux2014-aarch64:latest
```

Comment:
For reasons of reproducibilty, we prefer to pin to tags that are less likely to change.